app/src/main/res/layout/activity_add_beacon_to_group.xml

```diff
@@ -0,0 +1,150 @@
+                    android:layout_toEndOf="@id/imageview_add_person"
+                    android:layout_toStartOf="@id/imagebutton_search_for_beacons"
+                    android:layout_alignParentTop="true"
+
+
+                    >
+                <EditText
+                        android:id="@+id/edittext_create_search_for_beacons"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:hint="@string/search_for_beacon"
+                        android:imeActionId="@+id/login"
+                        android:imeActionLabel="@string/action_sign_in_short"
+                        android:imeOptions="actionUnspecified"
+                        android:inputType="textAutoComplete"
+                        android:maxLines="1"
+                        />
+                </android.support.design.widget.TextInputLayout>
+
+                <ImageButton
+                    android:id="@+id/imagebutton_search_for_beacons"
+                    style="?android:borderlessButtonStyle"
+                    android:layout_width="48dp"
+                    android:layout_height="48dp"
+                    android:layout_alignParentEnd="true"
+                    android:src="@drawable/ic_search_accentcolor"
+
+                    />
+
+
+                <ListView
+                    android:id="@+id/listview_beacon_search_result"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:layout_below="@+id/textinputlayout_search_for_beacons"
+
+                    />
+            </RelativeLayout>
+
+        </android.support.v7.widget.CardView>
+
+        <android.support.v7.widget.CardView
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:layout_weight="0.5"
+            android:layout_marginTop="8dp"
+            android:layout_marginBottom="8dp"
+            >
+            <RelativeLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:padding="8dp"
+                >
+                <TextView
+                    android:id="@+id/textview_added_beacons"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="@string/added_beacons"
+                    android:fontFamily="sans-serif-light"
+                    android:textSize="22sp"
+                    />
+
+                <ListView
+                    android:id="@+id/listview_added_beacons"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+
+                    android:layout_below="@+id/textview_added_beacons"
+                    android:layout_above="@id/btn_accept_searched_beacons"
```

Comment:
`Error:(128, 43) No resource found that matches the given name (at 'layout_above' with value '@id/btn_accept_searched_beacons').`